pkgs/development/libraries/gdk-pixbuf/setup-hook.sh

```diff
@@ -1,12 +1,9 @@
 findGdkPixbufLoaders() {
 
-    # choose the longest loaders.cache
     local loadersCache="$1/lib/gdk-pixbuf-2.0/2.10.0/loaders.cache"
     if [[ -f "$loadersCache" ]]; then
         if [[ -f "${GDK_PIXBUF_MODULE_FILE-}" ]]; then
-            if (( "$(cat "$loadersCache" | wc -l)" > "$(cat "$GDK_PIXBUF_MODULE_FILE" | wc -l)" )); then
-                export GDK_PIXBUF_MODULE_FILE="$loadersCache"
-            fi
+            export GDK_PIXBUF_MODULE_FILE="$loadersCache":$GDK_PIXBUF_MODULE_FILE
         else
             export GDK_PIXBUF_MODULE_FILE="$loadersCache"
         fi
```

Comment:
This will clear the variable e.g. when `GDK_PIXBUF_MODULE_FILE` already contains multiple values. Just switch it to

```
addToSearchPath GDK_PIXBUF_MODULE_FILE "$loadersCache"
```